src/dataFunctions.js

```diff
@@ -1,9 +1,18 @@
-// Estas funciones son ejemplos, aquí puedes desarrollar tus propias funciones.
+export const sortData = (data, sortBy, sortOrder) => {
+  const sortOrderFactor = sortOrder === 'desc' ? -1 : 1;
 
-export const example = () => {
-  return 'example';
-};
-
-export const anotherExample = () => {
-  return [];
+  if (sortBy && sortOrder!== 'default') {
```

Comment:
Would you like add 'default' as a default value of parameter? 
maybe you no need here, only reference.
```suggestion
  if (sortBy && !sortOrder) {
```